bit-rot: simplify and cleanup time management

dmantipov opened the pull request:
bit-rot: simplify and cleanup time management
    
Drop trivial `br_fsscan_calculate_delta()`, consistently use
`unsigned long` and `time_t` to represent time intervals here
and there, drop unused `tv` from `struct br_child`, adjust
related code.
    
Signed-off-by: Dmitry Antipov <dantipov@cloudlinux.com>
Updates: #1000

dmantipov commented:
/run regression

gluster-ant commented:
0 test(s) failed 


1 test(s) generated core 
./tests/bitrot/br-signer-threads-config-1797869.t

2 test(s) needed retry 
./tests/000-flaky/basic_afr_split-brain-favorite-child-policy.t
./tests/basic/ec/self-heal.t
https://build.gluster.org/job/gh_centos7-regression/2227/

dmantipov commented:
/run regression

gluster-ant commented:
0 test(s) failed 


1 test(s) generated core 
./tests/bitrot/br-signer-threads-config-1797869.t

3 test(s) needed retry 
./tests/000-flaky/basic_afr_split-brain-favorite-child-policy.t
./tests/000-flaky/basic_mount-nfs-auth.t
./tests/000-flaky/bugs_nfs_bug-1116503.t

2 flaky test(s) marked as success even though they failed 
./tests/000-flaky/basic_mount-nfs-auth.t
./tests/000-flaky/bugs_nfs_bug-1116503.t
https://build.gluster.org/job/gh_centos7-regression/2229/

Commit:
bit-rot: simplify and cleanup time management

Drop trivial 'br_fsscan_calculate_delta()', consistently use
'unsigned long' and 'time_t' to represent time intervals here
and there, drop unused 'tv' from 'struct br_child', adjust
related code.

Signed-off-by: Dmitry Antipov <dantipov@cloudlinux.com>
Updates: #1000

dmantipov commented:
/run regression
